### PR TITLE
[Improve] DES cryptographic algorithm change to AES

### LINKED ISSUE #2241: [Bug] DES cryptographic algorithm is weak and should not be used.
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-streampark/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.

### What happened

The DES encryption algorithm is used in EncryptUtils, which is an older algorithm that is now considered weak. should be use AES  encryption algorithm, which is a strong modern algorithm.

### StreamPark Version

all

### Java Version

all

### Flink Version

all

### Scala Version of Flink

all

### Error Exception

```log
none
```

### Screenshots

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by wolfboys**
## DES cryptographic algorithm change to AES

The DES encryption algorithm is used in EncryptUtils, which is an older algorithm that is now considered weak. should be use AES encryption algorithm, which is a strong modern algorithm.

## What changes were proposed in this pull request

Issue Number: close #2241

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

## Does this pull request potentially affect one of the following parts
 - Dependencies (does it add or upgrade a dependency): (yes / no)

### CODE REVIEW

**Comment by wolfboys** [comment]
cc @1996fanrui PTAL, thanks

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/util/EncryptUtils.java [hunk 1]
```diff
@@ -17,86 +17,64 @@
...
   }
 
-  private Key getKey(byte[] arrBTmp) {
-    byte[] arrB = new byte[8];
-    for (int i = 0; i < arrBTmp.length && i < arrB.length; i++) {
-      arrB[i] = arrBTmp[i];
-    }
-    return new javax.crypto.spec.SecretKeySpec(arrB, "DES");
+  public static byte[] base64Decode(String base64Code) {
+    return Base64.getDecoder().decode(base64Code);
```
Comment (1996fanrui) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/util/EncryptUtils.java]:
```suggestion
  private static byte[] base64Decode(String context) {
    return Base64.getDecoder().decode(context);
```

File: streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/util/EncryptUtils.java [hunk 2]
```diff
@@ -17,86 +17,64 @@
...
+    return cipher;
   }
 
-  String decrypt(String strIn) {
-    try {
-      return new String(decrypt(hexStr2ByteArr(strIn)));
-    } catch (Exception e) {
-      return "";
-    }
+  public static String base64Encode(byte[] bytes) {
```
Comment (1996fanrui) [inline streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/base/util/EncryptUtils.java]:
```suggestion
  private static String base64Encode(byte[] bytes) {
```

**Review by 1996fanrui — COMMENTED** [review]
@wolfboys , thanks for your contribution, I have left some comments, please take a look in your free time, thanks!

**Comment by wolfboys** [comment]
> @wolfboys , thanks for your contribution, I have left some comments, please take a look in your free time, thanks!

thanks for your review. done.

**Review by 1996fanrui — APPROVED** [review]
Thanks for the update, LGTM.

**Pull request merged**